src/elude_tool/data/unit_tests/UnitTest_Elude_DataManager.cpp

```diff
@@ -27,14 +27,16 @@
 #include "Globals.h"
 #include "Enzyme.h"
 
+#define PATH_TO_DATA string("")
```

Comment:
These definitions should not be necessary, PATH_TO_DATA and PATH_TO_WRITABLE should be set by CMake.

Reply:
This was the change by cmake (I haven't actually edited this file), these changes can be ignored during a merge